### DISCUSSION TITLE
PR not updated after I pushed commits to the branch

### DISCUSSION THREAD

**Discussion by Jianrong-Yu** [community/community#57447] [Pull Requests]
### Select Topic Area

Bug

### Body

Happens from this morning.
I tried to push commits to branches of  two PRs in two different repo, the PR is not updated for the new commits.
I verified the branch in code tab (https://github.com/xxx/xxx/tree/branch-name), the new commit DO exists.

**Comment by cvm-a** [comment]
I am also seeing the same issue, this happened for multiple PRs. on one of them, it updated after ~ 10 mins

**Reply by Jianrong-Yu** [reply]
I found one of them is updated after 20 mins.

**Comment by adamlui** [comment]
https://status.github.com shows it is degraded

![image](https://github.com/community/community/assets/10906554/5111f94f-6bdf-465b-8bf1-adca4116134b)

(Also Actions is down but not shown as such)

**Comment by Jianrong-Yu** [comment]
Seems it's a known issue, I'll close this one.
